tests/integration/TEC/Tickets/Order_Modifiers/Fees/Checkout/Checkout_Test.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace TEC\Tickets\Tests\Integration\Order_Modifiers\Fees\Checkout;
+
+use Codeception\TestCase\WPTestCase;
+use TEC\Tickets\Commerce\Cart;
+use TEC\Tickets\Commerce\Cart\Unmanaged_Cart;
+use TEC\Tickets\Commerce\Module as Commerce;
+use TEC\Tickets\Commerce\Order_Modifiers\Models\Fee;
+use TEC\Tickets\Commerce\Order_Modifiers\Models\Order_Modifier_Meta;
+use TEC\Tickets\Commerce\Order_Modifiers\Repositories\Order_Modifiers_Meta as Repository;
+use TEC\Tickets\Commerce\Order_Modifiers\Values\Float_Value;
+use Tribe\Tickets\Test\Commerce\Ticket_Maker;
+
+class Checkout_Test extends WPTestCase {
+
+	use Ticket_Maker;
+
+	/** @var int */
+	protected int $ticket_id;
+
+	/** @var int */
+	protected int $base_price = 15;
+
+	/** @var ?Unmanaged_Cart */
+	protected ?Unmanaged_Cart $cart;
+
+	/** @var ?Repository */
+	protected ?Repository $repository;
+
+	/**
+	 * @before
+	 */
+	public function set_up() {
+		$post_id         = static::factory()->post->create( [ 'post_type' => 'page' ] );
+		$this->ticket_id = $this->create_ticket( Commerce::class, $post_id, $this->base_price );
```

Comment:
Dont modify the DB state outside of a testcase.

You can modify the DB state inside data providers, which they do so inside a closure which is being called inside a testcase.

So please create post and ticket either directly inside a tescase or in a provider's closure.